src/utils/polygon.cpp

```diff
@@ -1138,6 +1166,39 @@ Polygons Polygons::smooth_outward(const AngleDegrees max_angle, int shortcut_len
 }
 
 
+
+
+void ConstPolygonRef::splitPolylineIntoSegments(Polygons& result) const 
+{
+    Point last = front();
```

Comment:
One interpretation of 'last' is the same as 'previous'. "next before the present : most recent." However, 'last' also has a different meaning, which could cause some confusion.  The code here is super simple, though.

I'm okay with the suggested change, but the use of the word 'last' in that way is scattered throughout the engine. It could make sense to adjust it everywhere, but maybe not in this PR.